wsgi/config.py

```diff
@@ -1,21 +1,34 @@
 # see http://docs.gunicorn.org/en/latest/configure.html#configuration-file
 
 from os import getenv
+from multiprocessing import cpu_count
 
 from prometheus_client import multiprocess
 
 
 bind = f'0.0.0.0:{getenv("PORT", "8000")}'
-workers = getenv('WEB_CONCURRENCY', 2)
-accesslog = '-'
-errorlog = '-'
-loglevel = getenv('LOGLEVEL', 'info')
+
+try:
+    cpu = cpu_count() * 2 + 1
+    workers = getenv("WEB_CONCURRENCY", cpu)
+except NotImplementedError:
+    workers = 3
+
+accesslog = "-"
+errorlog = "-"
+loglevel = getenv("LOGLEVEL", "info")
 
 # Larger keep-alive values maybe needed when directly talking to ELBs
 # See https://github.com/benoitc/gunicorn/issues/1194
-keepalive = getenv('WSGI_KEEP_ALIVE', 2)
-worker_class = getenv('GUNICORN_WORKER_CLASS', 'meinheld.gmeinheld.MeinheldWorker')
-worker_tmp_dir = '/dev/shm'
+keepalive = getenv("WSGI_KEEP_ALIVE", 118)
+worker_class = getenv("GUNICORN_WORKER_CLASS", "meinheld.gmeinheld.MeinheldWorker")
```

Comment:
Should we change this default to "gevent" now?

Reply:
Yeah, we should 'someday' but I wanted to be able to do it incrementally so I thought it made sense to stay as meinheld, do the full release cycle then come back and update here.